cats/cats-redis/src/main/java/com/netflix/spinnaker/cats/redis/cluster/ClusteredAgentScheduler.java

```diff
@@ -152,21 +157,28 @@ public void unschedule(Agent agent) {
 
     private static class AgentJob implements Runnable {
         private final long lockReleaseTime;
+        private final long lockErrorReleaseTime;
         private final AgentExecutionAction action;
         private final ClusteredAgentScheduler scheduler;
 
-        public AgentJob(long lockReleaseTime, AgentExecutionAction action, ClusteredAgentScheduler scheduler) {
-            this.lockReleaseTime = lockReleaseTime;
+        public AgentJob(Long[] times, AgentExecutionAction action, ClusteredAgentScheduler scheduler) {
+            this.lockReleaseTime = times[0] + times[1];
```

Comment:
Maybe add another enum or const ints to give more context to what each index of times means without digging into the code further?